XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt

```diff
@@ -0,0 +1,19 @@
+<muxc:NavigationView x:Name="nvSample9" 
+                            Grid.Row="1" 
+                            Height="460"
```

Comment:
I'm tempted to remove this as well. It's an app-specific implementation detail that probably shouldn't be reflected in the sample code. We don't want app developers in general hardcoding heights like this.